docs/users/extend/custom-commands.md

```diff
@@ -154,7 +154,7 @@ The following fields can be used for a flag definition:
 * `Type`: possible values are `bool`, `string`, `int`, `uint` (defaults to bool)
 * `DefValue`: default value for usage message
 * `NoOptDefVal`: default value, if the flag is on the command line without any options
-* `Annotations`: used by cobra.Command bash autocomple code see <https://github.com/spf13/cobra/blob/master/bash_completions.md>
+* `Annotations`: used by cobra.Command bash autocomplete code see <https://github.com/spf13/cobra/blob/master/bash_completions.md>
```

Comment:
I don't understand how the spellcheck didn't catch this. I don't see an error in .spellcheckwordlist.txt though...